index.js

```diff
@@ -0,0 +1,15 @@
+const menuBtn = document.getElementById('menu-i');
+const closeBtn = document.getElementById('menu-i-close');
+const mobileMenu = document.querySelector('.mobile-nav');
+const menuItems = document.querySelectorAll('.mobile-nav ul li');
+const openMenu = () => mobileMenu.classList.add('open-menu');
+const closeMenu = () => mobileMenu.classList.remove('open-menu');
+menuItems.forEach((el) => {
+  el.addEventListener('click', () => closeMenu());
+});
+menuBtn.addEventListener('click', () => {
+  openMenu();
+});
+closeBtn.addEventListener('click', () => {
+  closeMenu();
+});
```

Comment:
- [ ] Please make sure when the user expands to desktop size the hamburger menu button is not visible. 

<img width="1099" alt="Screenshot 2023-05-22 at 10 34 14 PM" src="https://github.com/devendra-alt/portfolio/assets/44798044/78905985-415b-411c-b276-adba61c3a8a7">